mappings/services.json

```diff
@@ -21,19 +21,49 @@
           "type": "stemmer",
           "name": "possessive_english"
         }
+      },
+      "char_filter": {
+        "nonalpha_removal": {
+          "type": "pattern_replace",
+          "pattern": "\\W+",
+          "replacement": ""
+        }
+      },
+      "normalizer": {
+        "filter_normalizer": {
+          "type": "custom",
+          "char_filter": [
+            "nonalpha_removal"
+          ],
+          "filter": [
+            "lowercase"
+          ]
+        }
       }
     }
   },
   "mappings": {
     "services": {
       "_meta": {
+        "dmSortClause": [
```

Comment:
Why is this camel case? Everything else in the mapping is underscore-separated. Same with sortonly_serviceIdHash.

Reply:
Yeah, guess we should stick with the underscored notation for keys aimed at search api/ES consumption

Reply:
done